feature/account/setup/src/main/kotlin/app/k9mail/feature/account/setup/ui/common/AccountSetupTopAppBar.kt

```diff
@@ -0,0 +1,44 @@
+package app.k9mail.feature.account.setup.ui.common
+
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.res.stringResource
+import app.k9mail.core.ui.compose.common.DevicePreviews
+import app.k9mail.core.ui.compose.designsystem.organism.TopAppBar
+import app.k9mail.core.ui.compose.theme.K9Theme
+import app.k9mail.core.ui.compose.theme.MainTheme
+import app.k9mail.core.ui.compose.theme.ThunderbirdTheme
+import app.k9mail.feature.account.setup.R.string
+
+@Composable
+internal fun AccountSetupTopAppBar(
+    title: String,
+) {
+    TopAppBar(
+        title = title,
+        subtitle = stringResource(id = string.account_setup_top_bar_subtitle),
+        titleContentPadding = PaddingValues(
+            start = MainTheme.spacings.double,
```

Comment:
When there's no navigation icon in the toolbar, the title should start 16dp from the edge of the screen. It appears `MaterialTopAppBar` is automatically doing the right thing. Specifying `titleContentPadding` adds an additional 16dp, shifting the title/subtitle too far.

Reply:
This is according to design to align content and title. Could be a topic to the next design iteration.